### PR TITLE
Refresh Nu command docs for v0.101

### PR DESCRIPTION
**Pull request by hustcer**
Refresh Nu command docs for v0.101

### CODE REVIEW

File: .github/workflows/command-block.yml [hunk 1]
```diff
@@ -4,9 +4,16 @@ on:
     paths:
       - 'commands/docs/**'
 
+# Fix Resource not accessible by integration (updatePullRequest)
+permissions:
+  contents: write
```
Comment (sholderbach) [inline .github/workflows/command-block.yml:9]: Thanks for digging into why that workflow never commented!

Do we need the `contents` permissions, here? I think those are for modifying the repository itself or comments associated with commits for some reason.

File: .github/workflows/command-block.yml [hunk 2]
```diff
@@ -4,9 +4,16 @@ on:
...
+  pull-requests: write
+
 jobs:
   comment:
     runs-on: ubuntu-latest
+    # If the docs was updated by hustcer, skip the commenting step
+    if: ${{!contains(github.event.pull_request.user.login, 'hustcer') }}
```
Comment (sholderbach) [inline .github/workflows/command-block.yml:16]: Not security critical here but that would match all user names containing yours as a substring. either a direct match or an array of user names in the first arg of `contains` would make sense.

File: .github/workflows/command-block.yml (hunk 1 above)
Comment (hustcer) [reply .github/workflows/command-block.yml]: I have tried many ways, It works in my repo: https://github.com/hustcer/nushell.github.io/pull/3  however, it does not work here: https://github.com/nushell/nushell.github.io/actions/workflows/command-block.yml
I guess some settings in `Actions permissions` need to be changed and I don't have the permission @fdncred

Comment (sholderbach) [reply .github/workflows/command-block.yml]: Resolved by #1704 and following

**Pull request merged**